xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/edit.less

```diff
@@ -2,34 +2,51 @@
 // Edit
 // --------------------------------------------------
 
-
-.bottombuttons {
+.sticky-buttons {
   background-color: @xwiki-page-content-bg;
   .buttons {
     margin: 0;
     padding: 1em;
   }
 
-  &.bottombuttons-fixed {
+  &.sticky-buttons-fixed {
     z-index: 100;
     position: fixed;
     bottom: 0;
     background-color: @breadcrumb-bg;
+    @media (max-width: @screen-sm-max) {
+      #commentinput, #autosaveControl, .metamodifs {
+        display: none;
+      }
+    }
+  }
+
+  input[type=checkbox] {
```

Comment:
I would prefer to have the quotation marks, like [type="checkbox"], especially since the value is not numeric

Reply:
Also this selector is very generic, hope it won't affect multiple places in a negative way. Would be good if it would mark in a positive way :) Still if it is so generic, I think it would be better suited in forms.less than in edit.less

Reply:
Sure!

Reply:
> Also this selector is very generic, hope it won't affect multiple places in a negative way. 

It is only used inside the `.sticky-buttons` elements.